src/3.26/sass/widgets/_popovers-menus.scss

```diff
@@ -0,0 +1,153 @@
+/* 
+ * Popovers/Menus * 
+ */
+ 
+.popup-menu {
+  min-width: 200px;
+
+  .popup-menu-arrow { } //defined globally in the TOP BAR
+  .popup-sub-menu {
+    background-color: $alt_base_color;
+    box-shadow: $shadow_0;
+  }
+
+  .popup-menu-content { padding: 8px 0; }
+  .popup-menu-item {
+    // min-height: $menuitem_size - 4px * 2;
```

Comment:
Should this comment be removed?

Reply:
Yep! The GNOME Shell theme isn't currently linted and contains a lot of legacy code, so comments like this are pretty common. I'll get it removed.